demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -1,8 +1,84 @@
 from typing import List
 
+import pytest
+
 from demisto_sdk.commands.doc_reviewer.doc_reviewer import DocReviewer
 
 
+def doc_reviewer(
+    file_path: str,
+    known_words_file_path: str = None,
+    no_camel_case: bool = False,
+    no_failure: bool = False,
+    expand_dictionary: bool = False,
+    templates: bool = False,
+    use_git: bool = False,
+    prev_ver: str = None,
+    release_notes_only: bool = False
+) -> DocReviewer:
+
+    return DocReviewer(
+        file_path=file_path,
+        known_words_file_path=known_words_file_path,
+        no_camel_case=no_camel_case,
+        no_failure=no_failure,
+        expand_dictionary=expand_dictionary,
+        templates=templates,
+        use_git=use_git,
+        prev_ver=prev_ver,
+        release_notes_only=release_notes_only
+    )
```

Comment:
I don't think there's a real need for this, DocReviewer has default values for almost everything already or am I missing something here?

Reply:
its not that important, its just for better reading of the code, ill remove it.